release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -230,6 +230,19 @@ With the GA, you can enable descheduler profiles and configure the descheduler i
 
 For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[Evicting pods using the descheduler].
 
+[id="ocp-4-7-nodes-scheduler-profiles"]
+==== Scheduler profiles (Technology Preview)
+
+You can now specify a scheduler profile to control how pods are scheduled onto nodes. This is a replacement for configuring a scheduler policy. The following scheduler profiles are available:
+
+* `LowNodeUtilization`: This profile attempts to spread pods evenly across nodes to get low resource usage per node.
+
+* `HighNodeUtilization`: This profile attempts to place as many pods as possible on to as few nodes as possible, to minimize node count with high usage per node.
```

Comment:
s/on to/onto ? You use it as one word in the lead-in sentence. I think it could work either way but it would be clearer to be consistent.